.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "monthly"
```

Comment:
You may have been out but we talked about doing a 6-month interval for dependency upgrades as it gets to be a bit much monthly.

Reply:
I was there for that but I looked into it and unfortunately it appears that the only intervals supported by dependabot are:
daily, weekly, and monthly (unless I'm missing something).

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval

Reply:
Bummer